recipes/metplus/meta.yaml

```
@@ -0,0 +1,126 @@
+{% set version = "6.0.0" %}
+{% set met_version = "12.0.2" %}
+
+package:
+  name: metplus
+  version: {{ version }}
+
+source:
+  - url: https://github.com/dtcenter/METplus/archive/refs/tags/v{{ version }}.tar.gz
+    md5: a5612dfd025055707f4a45dff028e628
+    sha256: e9358aede2fd2abecd81806227de7b165d68fdf2fc9defcbba24df229461b155
+
+  - url: https://github.com/dtcenter/MET/archive/refs/tags/v{{ met_version }}.tar.gz
+    md5: 4b9dba80639a74f1a3ecb15357785603
+    sha256: 613c74cbe5a0afa87422e81a4cd70984238c1a0eaf1a395503486331223ede45
+    folder: MET
+
+  - url: https://downloads.sourceforge.net/project/gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/ghostscript-fonts-std-8.11.tar.gz
+    md5: 6865682b095f8c4500c54b285ff05ef6
+    sha256: 0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401
+    folder: gs-fonts
+
+
+build:
+  skip: True  # [win]
+  number: 0
+  entry_points:
+    - run_metplus.py = metplus.scripts.run_metplus:cli_main
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - python 3.10
+    - python-configuration
+
+  host:
+    - python 3.10
```

Use a `skip: true  # [py!=310]` instead.
```suggestion
    - python
```